Core/Utilities/MySQL.py

```diff
@@ -615,13 +614,13 @@ def _query(self, cmd, conn=None, debug=False):
       #  self.logger.debug('_query: %s ...' % str(res[:10]))
 
       retDict = S_OK(res)
-    except BaseException as x:
+    except Exception as x:
       # self.log.debug('_query: %s' % self._safeCmd(cmd))
-      retDict = self._except('_query', x, 'Execution failed.')
+      retDict = self._except('_query', x, 'Execution failed.', cmd)
```

Comment:
hum, you maybe don't want to log that, it may contain sensitive information. At the very least, use `_safeCmd`.

Reply:
Thanks, done.